types/react-loadable/webpack.d.ts

```diff
@@ -0,0 +1,23 @@
+import * as webpack from 'webpack';
+
+export namespace ReactLoadablePlugin {
+  interface Options {
+    filename: string;
+  }
+}
+
+export class ReactLoadablePlugin extends webpack.Plugin {
+  constructor(opts?: ReactLoadablePlugin.Options);
+}
+
+export interface Bundle {
+  id: number;
+  name: string;
+  file: string;
+}
+
+export interface Manifest {
+  [moduleId: string]: Bundle[];
+}
+
+export function getBundles(manifest: Manifest, moduleIds: string[]): Bundle[];
```

Comment:
This file does commonjs exports; can't use ES module syntax here. https://github.com/jamiebuilds/react-loadable/blob/master/src/webpack.js#L62-L63

```ts
const exports: {
  getBundles: typeof getBundles;
  ReactLoadablePlugin: typeof ReactLoadablePlugin;
}
export = exports
```